src/js/stores/account.js

```diff
@@ -60,8 +60,20 @@ export default Reflux.createStore({
   */
 
   onLogin(username, password, updatePayload) {
+    
     if (localStorage.getItem('webId')){
-      Account.login.completed(localStorage.getItem('webId')) 
+      
+      // Check if the cookie is still valid
+      fetch(`${proxy}/proxy?url=${localStorage.getItem('webId')}`, {
+        method: 'GET',
+        credentials: 'include'
+      }).then((res)=>{
```

Comment:
You could also use the structure

```
if (res.ok){
  do something
} else {
  do something else
}
```
res.ok basically returns true in case of 20x and false in case of 40x or something like that.